docs/content/en/docs/getting-started/optional/registrymirror.md

```diff
@@ -56,7 +56,7 @@ spec:
   ```yaml
   ociNamespaces:
     - registry: "public.ecr.aws"
-      namespace: "eks-anywhere"
```

Comment:
What is the implication of this? Will each registry mirror just use the default namespace? Or do you mean for them to choose their own name?

Reply:
Yea I thought this is still configurable, and maybe we just explain how this maps to our registries?